tools/koblingsplan/script.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+ 
+'''
+    Parse the exported table of tg23-koblingsplan (copypaste via libreoffice -> save as csv (standard values everywhere)) into a sensible yaml file
+    Will handle merged cells (e.g. keep previous iterations value if current iteration is empty)
+'''
+ 
+import csv
+import yaml
+ 
+# Holds all data. List of objects, each object represents a row in the table
+dataset = []
+
+with open('tg23-koblingsplan.csv', newline='') as csvfile:
+    csv_data = csv.reader(csvfile, delimiter=',', quotechar='"')
+ 
+    # for loop counter
+    i = 0
+ 
+    # Holds the data from the current iteration
+    current_iteration = {}
+ 
+    for row in csv_data:
```

Comment:
FYI, you can use enumerate to get the index, instead of creating an `i = 0` and add to it. 

```py
for i, row in enumerate(csv_data)
```